tests/unit/experimental/components/test_schema.py

```diff
@@ -794,12 +794,9 @@ def schema_json_with_nodes_without_labels() -> str:
                     {"name": "name", "type": "STRING"}
                 ]
             },
-            {
-                "label": "Organization",
-                "properties": [
-                    {"name": "name", "type": "STRING"}
-                ]
-            }
+            "Organization",
+            "",
+            "Company"
```

Comment:
Do not change anything but just our of curiosity: are the "Organization" and "Company" test cases actually testing different scenarios ?